### ISSUE TITLE
Access remote query endpoints using SPARQLStore

### ISSUE THREAD

**Issue by nicholsn** [incf-nidash/nidm-api#20]
rdflib provides an interface for using SPARQLStore and SPARQLUpdateStore interfaces to query remote sparql endpoints. 

 At some point it would be great to have the option to run queries over such remote stores, for example see: https://github.com/nicholsn/niquery/blob/master/niquery/query.py#L26

**Comment by vsoch** [comment]
yeah... that involves looking at sparql things... you can add that part :)

**Comment by nicholsn** [comment]
haha, figured as much =)
